.github/workflows/dotnet-core.yml

```diff
@@ -0,0 +1,22 @@
+name: RandomData Test Pipeline
```

Comment:
Why test? It's our default pipeline so maybe "RandomData Pipeline" or just "Github Pipeline". It's up to you.